apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff

```diff
@@ -829,11 +859,31 @@ SimpleX серверы не могут получить доступ к ваше
         <target>соединитесь с разработчиками.</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="connecting…" xml:space="preserve">
+        <source>connecting…</source>
+        <target>соединеняется…</target>
+        <note>chat list item title</note>
+      </trans-unit>
+      <trans-unit id="connection established" xml:space="preserve">
+        <source>connection established</source>
+        <target>соединение установлено</target>
+        <note>chat list item title (it should not be shown</note>
+      </trans-unit>
+      <trans-unit id="connection:%@" xml:space="preserve">
+        <source>connection:%@</source>
+        <target>connection:%@</target>
```

Comment:
```suggestion
        <target>соединение:%@</target>
```

Reply:
don't want to translate it, it's not in UI